certified-connectors/Plumsail Documents/apiDefinition.swagger.json

```diff
@@ -270,7 +270,7 @@
         "tags": [
           "ProcessesTriggers"
         ],
-        "operationId": "FlowV1ProcessesFlowTriggersByUserIdByProcessIdByHookIdDelete",
+        "operationId": "FlowV1ProcessesFlowTriggersByProcessIdByHookIdDelete",
```

Comment:
@RFlipper  The operation has been updated. This change breaks existing flows. Please confirm that there is no external user is going to be affecting with this change.

Reply:
The previous PR changed this but the fix was not published, this PR returns the correct value.